src/main/kotlin/com/atlassian/performance/tools/jiraactions/page/SingleSelect.kt

```diff
@@ -15,16 +16,24 @@ internal class SingleSelect(
     locator: By
 ) {
     private val parent: By = ByChained(locator, By.xpath(".."))
-    private val input = ByChained(parent, By.tagName("input"))
+    private val inputLocator = ByChained(parent, By.tagName("input"))
 
     fun select(value: String) {
         val inputElement = driver.wait(
             timeout = Duration.ofSeconds(8),
-            condition = elementToBeClickable(input)
+            condition = elementToBeClickable(inputLocator)
         )
         inputElement.click()
         inputElement.sendKeys(Keys.BACK_SPACE, value, Keys.TAB)
     }
+
+    fun select(picker: (Collection<String>) -> String) {
```

Comment:
Thanks for the hardening. Could you add it to the changelog?

- [ ] mention hardening in the changelog

Also, it'd be nice to have a JPERF ticket, where we could describe the symptom. This way when someone else inevitably encounters the same problem, they'll know it's fixed and where to bump to.
It's enough for the bug description to contain the stacktrace.

Reply:
It's actually a better fix for https://ecosystem.atlassian.net/browse/JPERF-298

Reply:
Noice, let's mention it's a fix for this ticket. The changelog diff is still empty.

Reply:
It's already in the changelog :-)

Reply:
Ooooh, sorry. I'm not used to [a long-running "Unreleased" section](https://github.com/atlassian/jira-actions/compare/release-3.10.0...master).